Components/images/images.css

```diff
@@ -0,0 +1,27 @@
+.image-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1.5rem;
+    cursor: pointer;
+    margin-bottom: 1.8rem;
+    flex-wrap: wrap;
+    box-shadow: 1px 1px 2px var(--shadow-color);
+    border-radius: 10px;
+    border: 2px solid var(--light-border-color);
+    background-color: var(--background-color);
+    height: auto;
+    width: 100%;
+}
+
+.responsive{
+    width: 100%;
+    max-width: 400px;
```

Comment:
### Don't add max-width on the image itself
What if your component library user wanted the image to grow more than 400px?
You should give max width 100% to the image and if you want to restrict this image from growing then add max-width: 400px on the image container.